lib/urbanopt/geojson.rb

```diff
@@ -57,7 +57,698 @@ def files_dir
       def doc_templates_dir
         return File.absolute_path(File.join(@root_dir, 'doc_templates'))
       end
-      
+
+
+      def get_multi_polygons(building_json)
```

Comment:
Move this and other free functions to another file, just make them free functions in the URBANopt::GeoJSON module, not class members